var/spack/repos/builtin/packages/py-pynucleus/package.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPynucleus(PythonPackage):
+    """PyNucleus is a finite element code that specifically targets nonlocal operators."""
+
+    homepage = "https://sandialabs.github.io/PyNucleus/index.html"
+    git = "https://github.com/sandialabs/PyNucleus.git"
+
+    maintainers("cgcgcg")
+
+    refs = ["master", "develop"]
+
+    for ref in refs:
+        version(ref, branch=ref)
+
+    depends_on("py-mpi4py@2.0.0:", type=("build", "link", "run"))
+    depends_on("py-cython", type="build")
```

Comment:
```suggestion
    depends_on("py-cython", type=("build", "run"))
```
Technically this isn't packaged correctly. It should be listed under `setup_requires`, not `install_requires`. But since it is listed under the latter, `pip check` would complain about a missing dependency if cython isn't listed as a run-time dep.